### DISCUSSION TITLE
[Feature] Storage template: add {{ext_normalized}} tag

### DISCUSSION THREAD

**Discussion by talan-z** [immich-app/immich#6049] [Feature Request]
### The feature

Hi all,

I am currently storing my pictures in subfolders by filetype.
I.e. Movies go into /mov/
png into /png/
jpg into /jpg/

I propose an additional tag {{ext_normalized}} for storage template that only returns **lower case** characters.
Currently I have folders named JPG and another one jpg.

In addition, it should return "jpg" for "jpeg" ending as well

Thank you for the consideration!

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
Added in  #16667
